### ISSUE TITLE
Permission error

### ISSUE THREAD

**Issue by kennybix** [CMA-ES/pycma#151]
I got the follow error:
Exception has occurred: PermissionError
[Errno 13] Permission denied: 'C:\\Users\\User1\\outcmaes\\\\._mu_conditions.logdata'

after running the following code block
```
xopts, es = cma.evolution_strategy.fmin_con(obj_func, xk, 1.0, h=lambda xk:np.array(constraint_func(xk)), \
       options={'bounds':new_bounds, 'CMA_stds':xk},restarts=restarts)
```

please how do I fix this?

**Comment by nikohansen** [comment]
It looks like a bug in the path construction. This should be fixed in the current master branch by commit 59a3c95 (which can be installed like `pip install git+https://github.com/CMA-ES/pycma.git@master`).

**Comment by kennybix** [comment]
Thank you
